### PR TITLE
List chats in which the bot is active

### PR DESCRIPTION
**Pull request by martijnjanssen**

### CODE REVIEW

File: F1ScheduleTelegramBot/main.py [hunk 1]
```diff
@@ -165,6 +165,29 @@ async def list_schedule(update: Update, context: ContextTypes.DEFAULT_TYPE) -> N
     )
 
 
+async def list_chats(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    logging.info("Received chats command from user: {}".format(update.effective_chat.id))
```
Comment (skamoen) [inline F1ScheduleTelegramBot/main.py:169]: Technically, chat_id is not a user right? You can also get a user ID from the update if you want...

File: F1ScheduleTelegramBot/main.py [hunk 2]
```diff
@@ -165,6 +165,29 @@ async def list_schedule(update: Update, context: ContextTypes.DEFAULT_TYPE) -> N
...
+    logging.debug("Chat id dev: {} equals user chat_id: {}".format(
+        chat_dev[0],
+        update.effective_chat.id == chat_dev[0])
+    )
+    if update.effective_message.chat_id != int(chat_dev[0]):
+        return
+
+    message = "Registered chats: \n"
+    for chat in database.list_chats(dbconn):
+        message += "{} ({}, <code>{}</code>)\n".format(chat[2], chat[1], chat[0])
```
Comment (skamoen) [inline F1ScheduleTelegramBot/main.py:182]: Maybe convert chat to an object? These [n] parameters are hard to read

**Review by skamoen — APPROVED** [review]

File: F1ScheduleTelegramBot/main.py (hunk 2 above)
Comment (Fastjur) [reply F1ScheduleTelegramBot/main.py:182]: Agreed, throw a nice serializer/deserializer on top of it so we get a nice clean object.

**Review by Fastjur — CHANGES_REQUESTED** [review]

File: F1ScheduleTelegramBot/main.py (hunk 2 above)
Comment (martijnjanssen) [reply F1ScheduleTelegramBot/main.py:172]: Filed #31

**Review by Fastjur — APPROVED** [review]

File: F1ScheduleTelegramBot/main.py (hunk 1 above)
Comment (martijnjanssen) [reply F1ScheduleTelegramBot/main.py]: True, although that log originally came from your keyboard 😏

**Pull request merged**